rust/worker/src/system/types.rs

```diff
@@ -231,6 +231,7 @@ impl<C: Component> ComponentHandle<C> {
         }
     }
 
+    #[cfg(test)]
```

Comment:
I don't think this is meant to be test-only exposed, its library code that should be available to use